Description: Unpin unicode-width dev-dependency
 Upstream has the dependency pinned for MSRV CI purposes, which aren't relevant for Debian
Forwarded: not-needed
Last-Update: 2025-11-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -60,4 +60 @@ version = "1.0"
 version = "0.9"
-
-[dev-dependencies.unicode-width]
-version = "=0.1.12"
